src/cryptography/hazmat/primitives/asymmetric/ec.py

```diff
@@ -128,6 +129,32 @@ def verify(self, signature, data, signature_algorithm):
         Verifies the signature of the data.
         """
 
+    @classmethod
+    def from_encoded_point(cls, curve, data, backend):
+        if not isinstance(curve, EllipticCurve):
+            raise TypeError("curve must be an EllipticCurve instance")
+
+        if not backend:
+            raise ValueError('Must provide backend to decode point')
+
+        byte_length = (curve.key_size + 7) // 8
```

Comment:
This is an argument for properly exposing group and degree of the curve as properties so we aren’t ambiguously relying on key_size to be the right value for this.

Reply:
This is not a blocker now, just a note to myself. Refs 
https://github.com/pyca/cryptography/issues/2452